app/controllers/alaveteli_pro/stripe_webhooks_controller.rb

```diff
@@ -2,10 +2,37 @@
 # Does not inherit from AlaveteliPro::BaseController because it doesn't need to
 class AlaveteliPro::StripeWebhooksController < ApplicationController
 
+  before_action :read_event_notification
+
   def receive
+    return unless @stripe_event
```

Comment:
From my testing this line doesn't do anything as `@stripe_event` is never empty because Stripe gem always returns a `Stripe::Event` object

Reply:
ugh, it was intended to be blank if the sig didn't match - will double-check (sounds like I made an assumption)

Reply:
(d'oh - the `before_action` returns and that code is never reached - good catch, thanks!)